docs/src/modules/components/Link.tsx

```diff
@@ -61,7 +61,7 @@ const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(function Link(props,
   const router = useRouter();
   const pathname = typeof href === 'string' ? href : href.pathname;
   const className = clsx(classNameProps, {
-    [activeClassName]: router.pathname === pathname && activeClassName,
+    [activeClassName]: router && router.pathname === pathname && activeClassName,
```

Comment:
When is the router falsey here?

Reply:
I got the error that the router is `null` after added `Link` to `NewStartScreen` component. I think it is because of this.

<img width="902" alt="Screen Shot 2564-09-20 at 18 54 14" src="https://user-images.githubusercontent.com/18292247/133997943-2ec77de0-08cc-4910-bba7-f6ef3474a248.png">

I don't think there is no need to use `ReactDOMServer.renderToStaticMarkup` right? cc @hbjORbj

Reply:
`dropDown.appendChild(<NewStartScreen />` won't work because NewStartScreen is a JSX element.

Reply:
I have changed to simple `NextLink` instead and revert the change in `Link.tsx`.